ui/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "codegen": "openapi-rq -i http://localhost:8080/swagger-ui/api.json -o src/api"
+    "codegen": "openapi-rq -i http://localhost:${ORT_SERVER_CORE_PORT:-8080}/swagger-ui/api.json -o src/api"
```

Comment:
FYI, there are sill the following occurrences of `localhost:8080` in the code base, but I'm unsure which of these should probably also get addressed as part of this commit:

```shell
$ git grep localhost:8080
README.md:| ORT Server API | http://localhost:8080/swagger-ui |                                                         |
config/github/src/test/kotlin/GitHubConfigFileProviderFactoryTest.kt:            GitHubConfigFileProvider.GITHUB_API_URL to "http://localhost:8080",
docker-compose.yml:      test: [ "CMD-SHELL", "wget --no-verbose --tries=1 http://localhost:8080/api/v1/liveness" ]
scripts/docker/keycloak/HealthCheck.java:        String url = "http://localhost:8080/health/live";
scripts/docker/keycloak/master-realm.json:    "redirectUris" : [ "http://localhost:8081/*", "http://localhost:8080/*" ],
scripts/requests/env/http-client.env.json:    "host": "http://localhost:8080/api/v1",
transport/kubernetes/src/test/resources/kubeconfig:    server: localhost:8080
ui/src/api/requests/core/OpenAPI.ts:    BASE: "http://localhost:8080",
ui/src/components/providers.tsx:OpenAPIConfig.BASE = import.meta.env.VITE_API_URL || 'http://localhost:8080';
workers/reporter/src/main/resources/application.conf:  reportDownloadLinkPrefix = "http://localhost:8080"
```